doc/user_guide/user_guide.md

```diff
@@ -114,16 +114,17 @@ There are two ways to install the language container: (1) using a python script
      or the [bucketfs-client](https://github.com/exasol/bucketfs-client). 
      The following command uploads a given container into BucketFS through curl 
      command, an http(s) client: 
-      ```buildoutcfg
+      ```shell
       curl -vX PUT -T \ 
           "<CONTAINER_FILE>" 
-          "http://w:<BUCKETFS_WRITE_PASS>@$bucketfs_host:<BUCKETFS_PASS>/<BUCKETFS_NAME>/<PATH_IN_BUCKET><CONTAINER_FILE>"
+          "http://w:<BUCKETFS_WRITE_PASSWORD>@$bucketfs_host:<BUCKETFS_PORT>/<BUCKETFS_NAME>/<PATH_IN_BUCKET><CONTAINER_FILE>"
+      ```
+
+      Please note that specifying the password on command line will make your shell record the password in the history. To avoid leaking your password please consider to set an environment variable. The following examples sets environment variable `bucketfs_write_password`:
```

Comment:
```suggestion
      Please note that specifying the password on command line will make your shell record the password in the history. To avoid leaking your password please consider to set an environment variable. The following examples sets environment variable `BUCKETFS_WRITE_PASSWORD`:
```